gazebo_ros2_control_bolt/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ link_directories(
 
 # Libraries
 add_library(${PROJECT_NAME} SHARED
-  src/gazebo_ros2_control_plugin.cpp
+  src/gazebo_bolt_ros2_control_plugin.cpp
```

Comment:
This change has already been done in master.
Please provide a commit based on the current HEAD of master.